test/Microsoft.ML.CpuMath.PerformanceTests/AvxPerformanceTests.cs

```diff
@@ -0,0 +1,232 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Running;
+using Microsoft.ML.Runtime.Internal.CpuMath;
+
+namespace Microsoft.ML.CpuMath.PerformanceTests
+{
+    public class AvxPerformanceTests
+    {
+        private const int EXP_MAX = 127;
```

Comment:
A lot of this appears to be duplicated with the SsePerformanceTests.  Can we refactor the two classes so there is less duplication?